wsmaster/codenvy-hosted-api-admin/src/main/java/com/codenvy/api/user/server/AdminUserService.java

```diff
@@ -91,4 +92,40 @@ public Response getAll(@ApiParam(value = "Max items") @QueryParam("maxItems") @D
             throw new BadRequestException(e.getMessage());
         }
     }
+
+    @GET
+    @Path("/find")
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(value = "Gets all users whose e-mail/name contains(case insensitive) given part")
+    @ApiResponses({@ApiResponse(code = 200, message = "The response contains searching result"),
+                   @ApiResponse(code = 400, message = "Missed required parameters, parameters are not valid"),
+                   @ApiResponse(code = 500, message = "Internal server error occurred")})
+    public Response find(@ApiParam("User e-mail fragment")
```

Comment:
I guess that in this case it is not critical, because user do not provide to many fields to search by, but for me it's ok to user DTO class instead of query parameters.